src/logic/map_objects/tribes/tribe_descr.cc

```diff
@@ -128,11 +129,8 @@ void load_helptexts(Widelands::MapObjectDescr* descr,
 	} else {
 		log_warn("No helptext defined for '%s'", descr->name().c_str());
 	}
-	descr->set_helptexts(tribe_name, helptexts);
+	descr->set_helptexts(name(), helptexts);
```

Comment:
This function used to be static and received a tribe's name as a parameter. There is no reason for this design, so I converted it to a member function and use name() instead of the (now removed) parameter.